IBAnimatable/AnimatablePresentatedViewController.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  Created by Tom Baranes on 16/07/16.
+//  Copyright © 2016 Jake Lin. All rights reserved.
+//
+
+import UIKit
+
+public class AnimatablePresentedViewController: UIViewController, PresentationDesignable {
```

Comment:
In fact, I used `Presented` because Apple uses `Presented`, but I'm joining you to say that it can be confusing. I'm not fan of `AnimatableModalViewController` since it's restrictive, but there definitely a better name to find.